bug: config rule is preventing upgrade of @typescript-eslint/eslint-plugin from 5.62.0 to 7.5.0

see https://github.com/ipfs/aegir/actions/runs/8521375637/job/23339383672?pr=1492#step:5:11

& https://github.com/ipfs/eslint-config-ipfs/pull/211

updating all deps results in :

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: eslint-config-ipfs@6.0.1
npm ERR! Found: @typescript-eslint/eslint-plugin@7.5.0
npm ERR! node_modules/@typescript-eslint/eslint-plugin
npm ERR!   @typescript-eslint/eslint-plugin@"^7.5.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @typescript-eslint/eslint-plugin@"^5.52.0" from eslint-config-standard-with-typescript@37.0.0
npm ERR! node_modules/eslint-config-standard-with-typescript
npm ERR!   eslint-config-standard-with-typescript@"^37.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/sgtpooki/.npm/_logs/2024-04-03T01_19_59_735Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sgtpooki/.npm/_logs/2024-04-03T01_19_59_735Z-debug-0.log
```

and https://www.npmjs.com/package/eslint-config-standard-with-typescript says it's deprecated and to use https://www.npmjs.com/package/eslint-config-love

--- a/package.json
+++ b/package.json
@@ -128,7 +128,7 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^7.5.0",
     "@typescript-eslint/parser": "^7.5.0",
-    "eslint-config-love": "^44.0.0",
+    "eslint-config-love": "^47.0.0",
     "eslint-config-standard": "^17.1.0",
     "eslint-plugin-etc": "^2.0.3",
     "eslint-plugin-import": "^2.29.1",

--- a/package.json
+++ b/package.json
@@ -122,20 +122,20 @@
     "release": "aegir release"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.23.0",
-    "@typescript-eslint/parser": "^5.23.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-config-standard-with-typescript": "^37.0.0",
-    "eslint-plugin-etc": "^2.0.2",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-jsdoc": "^46.4.4",
-    "eslint-plugin-no-only-tests": "^3.0.0",
+    "@typescript-eslint/eslint-plugin": "^7.5.0",
+    "@typescript-eslint/parser": "^7.5.0",
+    "eslint-config-love": "^44.0.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-etc": "^2.0.3",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsdoc": "^48.2.2",
+    "eslint-plugin-no-only-tests": "^3.1.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^6.0.0"
+    "eslint-plugin-promise": "^6.1.1"
   },
   "devDependencies": {
-    "aegir": "^42.2.2",
-    "eslint": "^8.15.0"
+    "aegir": "^42.2.5",
+    "eslint": "^8.57.0"
   },
   "peerDependenciesMeta": {
     "typescript": {

--- a/package.json
+++ b/package.json
@@ -26,7 +26,11 @@
     "dist"
   ],
   "eslintConfig": {
-    "extends": "."
+    "extends": "./src/index.js",
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    }
   },
   "release": {
     "branches": [
@@ -115,7 +119,7 @@
   },
   "scripts": {
     "clean": "aegir clean",
-    "lint": "aegir lint",
+    "lint": "eslint src",
     "dep-check": "aegir dep-check -i @typescript-eslint/eslint-plugin -i @typescript-eslint/parser",
     "test": "npm run test:node",
     "test:node": "aegir test -t node",

--- a/src/ts.js
+++ b/src/ts.js
@@ -8,7 +8,7 @@ module.exports = {
   },
   extends: [
     './js.js',
-    'standard-with-typescript'
+    'love'
   ],
   plugins: [
     'etc'

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,0 +1,4 @@
+{
+  "extends": "aegir/src/config/tsconfig.aegir.json",
+  "include": ["src"]
+}

